nginx.conf

```diff
@@ -19,20 +19,20 @@ server {
   
   rewrite ^/clientdev/(.*)$ /development/sdk-dev/$1$is_args$args redirect;
```

Comment:
For ones where the pattern ends in `(.*)`, my understanding is I didn't need to add the trailing slash in the destination's `$1$is_args$args` because either

a) there's nothing in the `(.*)` and so the slash before it becomes the trailing slash, or
b) whatever is in the `(.*)` already has a trailing slash, in which case we don't want to add another one, or
c) whatever is in the `(.*)` doesn't already has a trailing slash, in which case one will get added by the `/` prefix rule

Reply:
You’re right. But you can do `(.*?)/?$` then include the slash in the redirect. This would prevent it from being captured.